x/concentrated-liquidity/types/constants.go

```diff
@@ -27,8 +29,11 @@ const (
 )
 
 var (
-	MaxSpotPrice       = sdk.MustNewDecFromStr("100000000000000000000000000000000000000")
+	MaxSpotPrice = sdk.MustNewDecFromStr("100000000000000000000000000000000000000")
+	// TODO: remove when https://github.com/osmosis-labs/osmosis/issues/5726 is complete.
 	MinSpotPrice       = sdk.MustNewDecFromStr("0.000000000001") // 10^-12
+	MinSpotPriceBigDec = osmomath.BigDecFromSDKDec(MinSpotPrice)
+	MinSpotPriceV2     = osmomath.NewDecWithPrec(1, 30)
```

Comment:
How was this min spot price picked? Also, it looks like `MinSqrtPriceBigDec` below still derives from the regular min spot price – is this intentional?

Reply:
Yes, `MinSqrtPriceBigDec` refers to V1 (old min). It is temporary and will be removed on completion.

10^-30 is the minimum we can support given our `BigDec` precision of 36 and exponent at price one of -6.

So 30 + 6 = 36. If interested, happy to breakdown the reasons in more detail but this stems from how we do price-to-tick conversion. Workaround could be found but was deemed unnecessary since the current min should be enough.